diff --git a/node_modules/@cord-sdk/react/dist/mjs/index.js b/node_modules/@cord-sdk/react/dist/mjs/index.js
index 7821559..f6cc655 100644
--- a/node_modules/@cord-sdk/react/dist/mjs/index.js
+++ b/node_modules/@cord-sdk/react/dist/mjs/index.js
@@ -3783,7 +3783,7 @@ function withSDK(WrappedComponent) {
     const ComponentWithSDK = React.forwardRef((props, ref) => {
         const { sdk: cordSDK } = useContext(CordContext);
         if (!cordSDK) {
-            return null;
+            //return null;
         }
         return React.createElement(WrappedComponent, { ref: ref, ...props });
     });
